src/modules/bigquery.js

```diff
@@ -95,12 +112,15 @@ export default class BigQuery {
 
   getColumnMinMax = (columnName, dataset, tableName, token) => {
     const url = `https://bigquery.googleapis.com/bigquery/v2/projects/${dataset.dataProject}/queries`;
-    const query = `SELECT MIN(${columnName}) AS min, MAX(${columnName}) AS max FROM [${dataset.dataProject}.datarepo_${dataset.name}.${tableName}]`;
+    const query = `SELECT MIN(${columnName}) AS min, MAX(${columnName}) AS max FROM \`${dataset.dataProject}.datarepo_${dataset.name}.${tableName}\``;
 
     return axios
       .post(
         url,
-        { query },
+        {
+          query,
+          useLegacySql: false,
```

Comment:
is this new param required for the framework, for BQ, or for something else?

Reply:
this is required for BQ, the backtics around the table name are specific to their "standard SQL" implementation which is enabled by setting `useLegacySql` to false